### ISSUE TITLE
Api endpoint will come from environment file

### ISSUE THREAD

**Issue by muhammadawaisshaikh** [naeem2290/pokemon#4]
there should be an environment file from which base url of a backend server will be used, currently, hardcoded url is used in the service, which is a bad practice

**Comment by naeem2290** [comment]
Yes @muhammadawaisshaikh  this is an improvement, Actually I don't find this app to go to production and we have something environment specific here. Just one API needs to be integrated that's why I used that directly.

But yes that is something that can be improved.

**Comment by muhammadawaisshaikh** [comment]
Always consider test as a prod grade app, interviewer is considering your test as a full fledged product. But that’s ok
